src/main/java/kr/codesqaud/cafe/basic/Article.java

```diff
@@ -1,33 +1,40 @@
 package kr.codesqaud.cafe.basic;
 
+import java.sql.Timestamp;
+import java.time.format.DateTimeFormatter;
 
 public class Article {
     private int articleId;
     private String writer;
     private String title;
     private String contents;
 
+    private Timestamp timeStamp;
```

Comment:
```suggestion
    private Timestamp timeStamp;        
```
TimeStamp는 sql 에서 온 객체이기 때문에 일반적인 비지니스 로직을 표현하는 Article.java에서 사용하면 DB에 대한 강한 의존성이 생겨나서 추천하지 않는 것 같아요. 
그렇지만 사용을 했다면 DB의 타임스탬프 타입은 자동으로 현재시간을 넣어 줄 수 있기 때문에 굳이 데이터를 넣어 줄 필요는 없습니다.

Reply:
앗 그리고 지금 비어 있는 초록 스페이스 같은 게 보이는데 업무에서는 reject의 사유가 됩니다.